app/coffee/RealTimeRedisManager.coffee

```diff
@@ -1,6 +1,6 @@
 Settings = require('settings-sharelatex')
-rclient = require("redis-sharelatex").createClient(Settings.redis.realtime)
-Keys = Settings.redis.realtime.key_schema
+rclient = require("redis-sharelatex").createClient(Settings.redis.documentupdater)
+Keys = Settings.redis.documentupdater.key_schema
```

Comment:
At line 38 in this file there is a `rclient.publish "applied-ops", JSON.stringify(data)` -  this will publish on the `Settings.redis.documentupdater` rclient.  Presumably the intention is to change that to `Settings.redis.pubsub` after?

Reply:
I've actually pointed the publish over to pubsub now, I think I forgot to commit the second change I made. Cheers.